src/plugins/data/public/ui/dataset_selector/configurator.tsx

```diff
@@ -51,6 +52,17 @@ export const Configurator = ({
     return languages[0];
   });
 
+  const displayTimeFieldSelector = (ds: Dataset, lang: string, fields: DatasetField[]) => {
```

Comment:
`shouldDisplayTimeFieldSelector`? since calling this doesn't seem to display the selector